fe/fe-core/src/test/java/com/starrocks/sql/plan/ComplexTypePrunePlanTest.java

```diff
@@ -222,11 +223,11 @@ public void testLambda() throws Exception {
 
     @Test
     public void testCast() throws Exception {
-        String sql = "select cast(row(null, null, null) as STRUCT<a int, b MAP<int, int>, c ARRAY<INT>>); ";
+        String sql = "select cast(row(null, null, null) as STRUCT<`a` int, `b` MAP<int, int>, `c` ARRAY<INT>>); ";
```

Comment:
the change will make create view use sql：
```
SELECT CAST((row(NULL, NULL, NULL)) AS struct<`a` int(11), `b` map<int(11),int(11)>, `c` array<int(11)>>) AS `CAST((row(NULL, NULL, NULL)) AS struct<`a` int(11), `b` map<int(11),int(11)>, `c` array<int(11)>>)`
```
the alias wrap by `` ` ``, but it's conficit with `` `a` ``

Reply:
I think should check the print method of `Struct` in `AST2SQLBuilderVisitor`